content/cryptography/addresses/slides.md

```diff
@@ -114,18 +116,19 @@ _The first 5 words of the [BIP39 English dictionary](https://github.com/bitcoin/
 
 ## Mnemonic to Secret Key
 
-Of course, the secret key is a point on an elliptic curve, not a phrase.
-
-BIP39 applies 2,048 rounds of the SHA-512 hash function<br />to the mnemonic to derive a 64 byte key.
+The secret key is a scalar value from the scalar field of the base field which the elliptic curve is defined over. Not a phrase.
 
-Substrate uses the entropy byte array from the mnemonic.
+BIP39 applies 2,048 rounds of the SHA-512 hash function<br /> to the mnemonic to derive a 64 byte key.
 
 ---
 
 ## Portability
 
 Different key derivation functions affect the ability to use the same mnemonic in multiple wallets as different wallets may use different functions to derive the secret from the mnemonic.
 
+Notes:
+i.e. May hash to a different base field because of a different elliptic curve
```

Comment:
In here you can put the details previously added to the slide, saying that in ECC a secret key is actually a scalar value from the scalar field of the base field which the elliptic curve is defined over.